### DISCUSSION TITLE
How to enable legends for the graphs in the resource section.

### DISCUSSION THREAD

**Discussion by tumuvamsi** [hystax/optscale#491] [Q&A] [answered]
I would like to have resources names as legend some where on the graph which helps me to document it. Also is there any way to export the graphs ?.

**Comment by stanfra** [comment]
Hi @tumuvamsi, it's not very clear what you mean by "resources names as legend some where  on the graph", could you describe the required functionality more precisely? We currently do not have the ability to export the graphs.

**Reply by tumuvamsi** [reply]
I mean in the resource section , only if we place the cursor, it’s showing the  name of the resource . Can we show the names with color coding on the top or right side of the graph.

**Reply by stanfra** [reply]
We will discuss the possibility of adding a legend with the team and will keep you updated on the decision made.

**Reply by stanfra** [reply, marked as answer]
@tumuvamsi, legend functional  will be implemented in upcoming releases.
